workspaces/ui-v2/src/pages/onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,202 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Page } from '<src>/components';
+import { useAppSelector } from '<src>/store';
+import { useAppConfig } from '<src>/contexts/config/AppConfiguration';
+import { useAsyncMemo } from 'use-async-memo';
+import { Checkbox, Typography } from '@material-ui/core';
+import {
+  CapturesServiceContext,
+  useCaptures,
+} from '<src>/hooks/useCapturesHook';
+
+const useStyles = makeStyles((theme) => ({
+  body: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    width: '60%',
+    marginTop: theme.spacing(2),
+  },
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+interface OnboardingData {
+  specs_shared_with_team: number;
+  specs_shared_not_with_team: number;
+  specs_uploaded_by_gitbot: number;
+  coverages_uploaded: number;
+}
+
+export default function Onboarding() {
+  const classes = useStyles();
+
+  const specId = useAppSelector(
+    (state) => state.metadata.data?.specificationId!
+  );
+
+  const captures = useCaptures();
+  const capturesService = React.useContext(CapturesServiceContext);
+
+  const captureStatuses = useAsyncMemo(async () => {
```

Comment:
Can we move this into redux (and the onboarding fetched state)? It's not strictly necessary and isn't likely to be shared (side note, not everything needs to go into redux) - but it's a simple way for us to have separation of concerns. Right now, our component handles data fetching, data manipulation (loading, error, and transformation), and the UI components, this works and is fine but can make it more work to iterate on (lets say we add in more transformation, or fetch data from more sources - handling this can make this component more and more complex).

Redux: data fetching, manipulation, loading + error states
UI components: triggering data fetches, reading data + handling permutations of it (loading, error states, rendering of data)